StreamVideoTests/WebRTC/WebRTCClient_Tests.swift

```diff
@@ -641,7 +641,7 @@ final class WebRTCClient_Tests: StreamVideoTestCase {
         // Given
         let httpClient = HTTPClient_Mock()
         let response = Stream_Video_Sfu_Signal_UpdateMuteStatesResponse()
-        for _ in 0..<10 {
+        for _ in 0..<15 {
```

Comment:
Why did we increase this one?

Reply:
seems the sfu sends more trickles now and this sometimes fails.